source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.IntegrationTest.Core/GreenEnergyHub.Charges.IntegrationTest.Core.csproj

```
@@ -24,6 +24,10 @@ limitations under the License.
     <LangVersion>9</LangVersion>
     <OutputType>Library</OutputType>
   </PropertyGroup>
+
+  <ItemGroup>
+    <None Remove="TestFiles\Charges\PriceSeries\TariffPriceSeriesWithInvalidBusinessReasonCode.xml" />
+  </ItemGroup>
```

TariffPriceSeriesWithInvalidBusinessReasonCode.xml is also added in L104-106
```suggestion

  <ItemGroup>
    <None Remove="TestFiles\Charges\PriceSeries\TariffPriceSeriesWithInvalidBusinessReasonCode.xml" />
  </ItemGroup>
```